ci-operator/infra/openshift/release-controller/deploy-ocp-4.1.yaml

```diff
@@ -0,0 +1,294 @@
+    name: release-controller-modify
+  subjects:
+  - kind: ServiceAccount
+    namespace: ci
+    name: release-controller-ocp
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: release-controller-binding-ocp
+    namespace: openshift
+  roleRef:
+    kind: Role
+    name: release-controller-modify-ocp
+  subjects:
+  - kind: ServiceAccount
+    namespace: ci
+    name: release-controller-ocp
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: release-controller-binding-view
+    namespace: ocp
+  roleRef:
+    kind: ClusterRole
+    name: view
+  subjects:
+  - kind: ServiceAccount
+    namespace: ci
+    name: release-controller-ocp
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: release-controller-binding-prowjob-ocp
+    namespace: ci
+  roleRef:
+    kind: Role
+    name: release-controller-prowjob
+  subjects:
+  - kind: ServiceAccount
+    namespace: ci
+    name: release-controller-ocp
+
+# grant release-controller-ocp edit on its work namespace
+- kind: Namespace
+  apiVersion: v1
+  metadata:
+    name: ci-release
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: release-controller-binding-ocp
+    namespace: ci-release
+  roleRef:
+    kind: ClusterRole
+    name: edit
+  subjects:
+  - kind: ServiceAccount
+    namespace: ci
+    name: release-controller-ocp
+# give jobs in the work namespace authority to push to the
+# release namespace
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: release-controller-binding-promote
+    namespace: ocp
+  roleRef:
+    kind: ClusterRole
+    name: system:image-builder
+  subjects:
+  - kind: ServiceAccount
+    namespace: ci-release
+    name: builder
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: release-controller-binding-import
+    namespace: ocp
+  roleRef:
+    kind: Role
+    namespace: ocp
+    name: release-controller-import-ocp
+  subjects:
+  - kind: ServiceAccount
+    namespace: ci-release
+    name: builder
+
+# create the secret that holds the upgrade graph state
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: release-upgrade-graph
+    namespace: ocp
+
+# expose the files cache to the outside world
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: files-cache
+    namespace: ci-release
+  spec:
+    selector:
+      app: files-cache
+    ports:
+    - port: 80
+      targetPort: 8080
+- kind: Route
+  apiVersion: v1
+  metadata:
+    name: files-cache
+    namespace: ci-release
+  spec:
+    host: openshift-release-artifacts.svc.ci.openshift.org
+    tls:
+      termination: Edge
+      insecureEdgeTerminationPolicy: Redirect
+    to:
+      kind: Service
+      name: files-cache
+
+# deploy the controller
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: release-controller-ocp
+    namespace: ci
+- kind: Service
```

Comment:
you only need from here down

Reply:
Will remove, but why? Why is it different then 4.0?

Reply:
The items above are cluster-wide and don't need to be duplicated